### DISCUSSION TITLE
Variable passed to partial not displayed

### DISCUSSION THREAD

**Discussion by nicktc** [statamic/cms#4672] [Troubleshooting] [answered]
Hi,

My partial won't show the variable that was passed via the partial tag.

Home: home.antlers.html

`{{ partial:contact width="500" header="Some Title" }}`

Partial: _contact.antlers.html

```
Width: {{ width }}
Header: {{ header }}
```

Both are showing empty... According to the docs this is how it should be done. I am on Static version 3.2.20.

**Comment by jasonvarga** [comment]
Have you definitely given the view an `.antlers.html` extension?

It works for me when it's `.antlers.html`, but the variables are not rendered if I only have a `.html` extension.

**Comment by nicktc** [comment]
Yeah, I did. Here my partial file:
<img width="194" alt="Schermafbeelding 2021-11-11 om 14 24 35" src="https://user-images.githubusercontent.com/2048475/141305797-69a020c9-84ec-4c88-8c23-d1f46ecf9f33.png">

And the container:
<img width="148" alt="Schermafbeelding 2021-11-11 om 14 24 42" src="https://user-images.githubusercontent.com/2048475/141305837-b8528cb8-2e6f-419f-bd33-969321042856.png">

Almost feels like a bug. Can't imagine that I am doing something wrong (in this case). I tested this behaviour with multiple partials (contact partial, usecases partial, etc).

**Reply by jasonvarga** [reply]
Can you provide a repo that has this bug recreated?

In your initial post, you said you use the template `{{ partial:contact }}` yet you are showing `_usecases.antlers.html`.

Also, if it's a bug, feel free to just open an issue instead of a discussion.

**Reply by nicktc** [reply]
"I tested this behaviour with multiple partials (contact partial, usecases partial, etc)."

So you are right. The screenshot was from another partial. But they all live in the same directory, same naming convention, etc. But then the partial is called _contact.antlers.html.

I could create a new repo, but I am sure it will probably work (as it should) :). So then the question is; how or what should I debug to find why it's not working in my repo?

**Reply by nicktc** [reply]
I could even give you access to private repo to see the problem?

**Reply by nicktc** [reply]
@jasonvarga would you be able to checkout a private repo?

**Comment by nicktc** [comment]
@jackmcdade is this a bug, or am I doing something wrong?

**Comment by nicktc** [comment, marked as answer]
Resolved. Really now idea what was causing the problem, but now it suddenly worked :-). Sorry haha.
